JhipsterBlazor/Pages/Account/Register.razor

```diff
@@ -0,0 +1,100 @@
+                        </div>
+                    }
+                    @if (DoNotMatch)
+                    {
+                        <div class="alert alert-danger">
+                            The password and its confirmation do not match!
+                        </div>
+                    }
+                </div>
+            </div>
+
+            <div class="row justify-content-center">
+                <div class="col-md-8">
+                    <EditForm Model="@RegisterModel" OnValidSubmit="@HandleSubmit">
+                        <DataAnnotationsValidator />
+                        <div class="form-group">
+                            <label class="form-control-label" for="login">Username</label>
+                            <DynInputText id="login" @bind-Value="RegisterModel.Username" class="form-control" placeholder="Username" />
+                            <div class="form-text text-danger">
+                                <ValidationMessage For="@(() => RegisterModel.Username)" />
+                            </div>
+
+
+
+                        </div>
+
+                        <div class="form-group">
+                            <label class="form-control-label" for="email">Email</label>
+                            <DynInputText id="email" @bind-Value="RegisterModel.Email" type="email" class="form-control" placeholder="Email" />
+                            <div class="form-text text-danger">
+                                <ValidationMessage For="@(() => RegisterModel.Email)" />
+                            </div>
+
+                        </div>
+
+                        <div class="form-group">
+                            <label class="form-control-label" for="password">New password</label>
+                            <DynInputText id="password" @bind-Value="RegisterModel.Password" type="password" class="form-control" placeholder="New password" />
+                            <div class="form-text text-danger">
+                                <ValidationMessage For="@(() => RegisterModel.Password)" />
+                            </div>
+
+
+                            
+                        </div>
+
+                        
+
+
+                        <div class="form-group">
+                            <label class="form-control-label" for="confirmPassword">New password confirmation</label>
+                            <DynInputText id="confirmPassword" @bind-Value="RegisterModel.ConfirmPassword" type="password" class="form-control" placeholder="Confirm password" />
+                            <div class="form-text text-danger">
+                                <ValidationMessage For="@(() => RegisterModel.ConfirmPassword)" />
+                            </div>
+
+                        </div>
+
+                        <button type="submit" [disabled]="registerForm.invalid" class="btn btn-primary">Register</button>
```

Comment:
Can you remove [disabled] and convert to c# ?

Reply:
You have just remove disabled but you can add c# condition to check if form is valid and so activate or disable the button

Reply:
yeah I will implement it later, maybe with blazorise :)